normandy/recipes/management/commands/update_addon_urls.py

```diff
@@ -0,0 +1,51 @@
+from urllib.parse import urlparse, urlunparse
+from django.core.management.base import BaseCommand
+from django.template.defaultfilters import pluralize
+
+from normandy.recipes.models import RecipeRevision
+
+
+class Command(BaseCommand):
+    """
+    Rewrite all revisions to update the URL of add-ons stored in Normandy's
+    file storage to a new hostname.
+    """
+
+    help = "Updates add-on URL in revisions"
+
+    def add_arguments(self, parser):
+        parser.add_argument("new_hostname", help="Hostname to convert add-on urls to")
+
+    def handle(self, *args, new_hostname, **options):
+        target_revisions = RecipeRevision.objects.filter(action__name="opt-out-study")
+        update_count = 0
+        for rev in target_revisions.iterator():
```

Comment:
There's no need for this now. If there's only about ~2k records anyway, the iterator doesn't help. Not important though. I brought it because I wasn't sure how many of these there are in the DB and if it was a genuinely large number it'd help to use `.iterator()`.